HW_1/task_1.py

```diff
@@ -0,0 +1,13 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+n = int(input('Введите трехзначное число: '))
+
+c = n % 10
+n = n // 10
+b = n % 10
+a = n // 10 
```

Comment:
Без одной строчки можно обойтись.
```suggestion
c = n % 10
b = n // 10 % 10
a = n // 100
```